src/js/player.js

```diff
@@ -388,17 +391,7 @@ class Player extends Component {
     this.language(this.options_.language);
 
     // Update Supported Languages
-    if (options.languages) {
-      // Normalise player option languages to lowercase
-      const languagesToLower = {};
-
-      Object.getOwnPropertyNames(options.languages).forEach(function(name) {
-        languagesToLower[name.toLowerCase()] = options.languages[name];
-      });
-      this.languages_ = languagesToLower;
-    } else {
-      this.languages_ = Player.prototype.options_.languages;
-    }
+    this.languagetolowerif(options);
```

Comment:
I think this name may also have room for improvement (see the comment for pluginif()). Perhaps updateLanguages or something similar would be more informative based on the comment above this line.